docs/management/admin/trusted-apps.asciidoc

```diff
@@ -2,6 +2,8 @@
 [chapter, role="xpack"]
 = Trusted applications
 
+NOTE: To add Elastic Endpoint as a trusted application in your other antivirus (AV) software, refer to <<allowlist-endpoint-3rd-party-av-apps>>.
```

Comment:
A little extra context-setting might help readers understand _why_ they'd want to do this.
```suggestion
NOTE: If you use {elastic-defend} along with other antivirus (AV) software, you might need to configure the other system to trust {elastic-endpoint}. Refer to <<allowlist-endpoint-3rd-party-av-apps>> for more information.
```